Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -40,12 +40,15 @@ PTF_TEST_CASE(TestIPAddress)
 	PTF_ASSERT_EQUAL(ip4AddrFromIpAddr, secondIPv4Address);
 
 	pcpp::IPv4Address ipv4Addr("10.0.0.4"), subnet1("10.0.0.0"), subnet2("10.10.0.0"), mask("255.255.255.0");
+	std::string maskedSubnet1("10.0.0.0/24"), maskedSubnet2("10.10.0.0/24");
```

Comment:
I'd add a few negative tests when the subnet isn't valid, for example: `"aaaa"`, `"10.0.0.0"`, "`10.0.0.0/aa"`, `"10.0.0.0/33"`, `"999.999.1.1/24"`

Reply:
Done